src/test/java/com/google/sps/UserTest.java

```diff
@@ -1,6 +1,7 @@
 package com.google.sps;
 
 import com.google.sps.data.User;
+import com.google.sps.data.User.profileType.*;
```

Comment:
This looks like a coding artifact. Remove?

Reply:
done